Gem/Code/Source/Components/UI/HUDComponent.h

```diff
@@ -10,14 +10,20 @@
 
 #include <AzCore/Component/Component.h>
 #include <AzCore/EBus/Event.h>
+
+#if AZ_TRAIT_CLIENT
```

Comment:
you might e able to bring the Event.h header include in here

Reply:
Good call